packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -98,7 +106,292 @@ const experimentSchema = new mongoose.Schema({
   ideaSource: String,
 });
 
-export const ExperimentModel = mongoose.model<ExperimentDocument>(
+type ExperimentDocument = mongoose.Document & ExperimentInterface;
+
+const ExperimentModel = mongoose.model<ExperimentDocument>(
   "Experiment",
   experimentSchema
 );
+
+/**
+ * Convert the Mongo document to an ExperimentInterface, omitting Mongo default fields __v, _id
+ * @param doc
+ */
+const toInterface = (doc: ExperimentDocument): ExperimentInterface =>
+  omit(doc.toJSON(), ["__v", "_id"]);
+
+export async function getExperimentById(
+  id: string
+): Promise<ExperimentInterface | null> {
```

Comment:
The experiment id is not unique across organizations, so this needs to include the org id in the query